src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,2105 @@
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <param name="format">The command format</param>
+        /// <returns>The result status, true if supported</returns>
+        /// <exception cref="InvalidOperationException">This exception can be due to an invalid state.</exception>
+        /// <exception cref="ArgumentException">This exception can be due to an invalid parameter.</exception>
+        /// <pre>
+        /// The State must be Ready.
+        /// </pre>
+        public static bool IsCommandFormatSupported(CommandFormat format)
+        {
+            bool supported = false;
+            ErrorCode error = VcMgrIsCommandFormatSupported(format, out supported);
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "IsCommandFormatSupported Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+            return supported;
+        }
+
+        /// <summary>
+        /// Sets system or exclusive commands.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <param name="list">Command list</param>
+        /// <exception cref="InvalidOperationException">This exception can be due to invalid state.</exception>
+        /// <exception cref="ArgumentException">This exception can be due to an invalid parameter.</exception>
+        /// <pre>
+        /// The State should be Ready
+        /// </pre>
+        public static void SetCommandList(VoiceCommandList list)
+        {
+            if (list == null)
+                throw new ArgumentException("Parameter is null");
+
+            ErrorCode error = VcMgrSetCommandList(list._handle);
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "SetCommandList Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+        }
+
+        /// <summary>
+        /// Unsets command list.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <exception cref="InvalidOperationException">This exception can be due to invalid state.</exception>
+        /// <pre>
+        /// The State should be Ready
+        /// </pre>
+        public static void UnsetCommandList()
```

Comment:
How about  ClearCommands()?

Reply:
Csharp Apis were inherited from native c apis, so it would be better to use simillar signature. But if you think it doesn't matter, we can apply your recommendation.